govcd/lbserverpool.go

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServerPool creates a load balancer server pool based on mandatory fields. It is a synchronous
+// operation. It returns created object with all fields (including ID) populated or an error.
+// Name and Algorithm fields must be populated.
+func (eGW *EdgeGateway) CreateLBServerPool(lbPoolConfig *types.LBPool) (*types.LBPool, error) {
```

Comment:
Currently major our functions are held by hierarchy. So usually create belongs to parent and other functions to itself. Are we still follow that or we think it's OK loose a bit. @dataclouder @lvirbalas

Reply:
A discussion matter then - waiting for input.
To me - this avoids construction of new object and makes less code because all the operations are done on edge gateway (all load balancer components are children for edge gateway). Once you got a edge gateway, you don't need to construct any new objects.

Reply:
> usually create belongs to parent and other functions to itself. Are we still follow that or we think it's OK loose a bit. 

@vbauzysvmware I'm usually for consistency, but how consistent are we in this regard?
@dataclouder your preference?

Reply:
For mostly of the times we are. Though my natural habit is more like holding all functions in itself.

Reply:
After further investigation, I believe @Didainius may leave it where it is, because:

a. There are many create functions that are already in their .go files:
CreateDisk
CreateOrgVDCNetwork
...

b. Previous PR already passed this, so it would require refactoring:
CreateLBServiceMonitor is in lbservicemonitor.go

c. Modern IDE like GoLand shows all functions on a struct even if they are in different files.

Reply:
So end line. Use as you like it. I will do create also not in hierarchy way.